dubbo-common/src/main/java/org/apache/dubbo/common/threadpool/manager/DefaultExecutorRepository.java

```diff
@@ -352,7 +354,11 @@ public ScheduledExecutorService getSharedScheduledExecutor() {
     }
 
     private ExecutorService createExecutor(URL url) {
-        return (ExecutorService) extensionAccessor.getExtensionLoader(ThreadPool.class).getAdaptiveExtension().getExecutor(url);
+        ExecutorService executorService = (ExecutorService) ExtensionLoader.getExtensionLoader(ThreadPool.class).getAdaptiveExtension().getExecutor(url);
```

Comment:
The original code should be used here
`ExecutorService executorService = (ExecutorService) extensionAccessor.getExtensionLoader(ThreadPool.class).getAdaptiveExtension().getExecutor(url);`

Reply:
doen, thanks

Reply:
all ```ExtensionLoader.getExtensionLoader``` might be changed to  ```extensionAccessor.getExtensionLoader``` , otherwise it will throws exception at destroyAll.